install/v2plugin/Dockerfile

```diff
@@ -0,0 +1,21 @@
+# Docker v2plugin container with OVS / netplugin / netmaster 
+
+FROM ubuntu:16.04
+MAINTAINER Cisco Contiv (http://contiv.github.io/)
+
+RUN mkdir -p /run/docker/plugins
+RUN mkdir -p /etc/openvswitch
+RUN mkdir -p /var/run/contiv/log
+
+RUN apt-get update
+RUN apt-get --assume-yes  install openvswitch-switch
```

Comment:
We need to specify the ovs version here. Please check the latest install scripts. @neelimamukiri had added this.